app/utils/jwt.py

```diff
@@ -58,7 +58,7 @@ def get_subscription_payload(token: str) -> Union[dict, None]:
         if "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." in token:
```

Comment:
i think its better to use `startswith` instead of `in` for finding `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.`